src/pages/_app.tsx

```diff
@@ -31,9 +32,11 @@ function app({ Component, pageProps }: AppProps) {
   });
   return (
     <ChakraProvider theme={theme}>
-      <Analytics />
-      <SpeedInsights />
-      <Component {...pageProps} />
+      <EnvContext.Provider value={pageProps.envs}>
```

Comment:
Meant as a placeholder until we have finalized a state manager.

Reply:
We we anticipate this env context vars to be critical to a number of things in client side code, we should use react-query to fetch and sync its state with a mobx `EnvContextStore`.